tests/e2e/scenarios/iperf3-attached/resources/iperf3-workload/attached-connector.yml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: skupper.io/v2alpha1
+kind: AttachedConnector
+metadata:
+  name: iperf3
+spec:
+  siteNamespace: 4uhmh-iperf3-hub
```

Comment:
Same here?

Reply:
This file is rewrite using a template that includes the prefix at each run. I can remove the file. It will be recreated every time.

Reply:
Fixed by: b2d9116d